php/locale.php

```diff
@@ -5,12 +5,37 @@
 include_once 'config.php';
 
 if ($OF_USE_LOCALES) {
+    $locale = null;
     if (isset($_GET["lang"])) {
-        $locale = $_GET["lang"];
-        $_SESSION["locale"] = $locale;
-    } else {
-        $locale = $_SESSION["locale"] ?? "en_US";
+        $lang = $_GET["lang"];
+
+        // We're accepting a random lang parameter... We should probably check it actually exists!
+        if (locale_exists($dbh, $lang)) {
```

Comment:
Where was `$dbh` defined? We haven't included `pdo_db.php`.

Reply:
Yeah, it's not. Didn't test this well!